.github/workflows/ci.yml

```diff
@@ -33,15 +41,42 @@ jobs:
         uses: dtolnay/rust-toolchain@master
         with:
           toolchain: ${{ matrix.rust_version }}
+          components: llvm-tools-preview
 
       - name: Cache Rust dependencies
         uses: Swatinem/rust-cache@v2
 
-      - name: Run unit tests
-        run: cargo test --all-features --verbose
+      - name: Install cargo-llvm-cov
```

Comment:
Pretty sure I've figured out why CodeCov uploads were failing before and I'd like to restore this check before we lose too much ground.